src/pages/settings/Security/SecuritySettingsPage.js

```diff
@@ -0,0 +1,63 @@
+import _ from 'underscore';
+import React from 'react';
+import {View, ScrollView} from 'react-native';
+import HeaderWithCloseButton from '../../../components/HeaderWithCloseButton';
+import Navigation from '../../../libs/Navigation/Navigation';
+import ROUTES from '../../../ROUTES';
+import styles from '../../../styles/styles';
+import * as Expensicons from '../../../components/Icon/Expensicons';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import MenuItem from '../../../components/MenuItem';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+const AboutPage = (props) => {
+    const menuItems = [
```

Comment:
We will imminently be adding to this array. Close account will be added with [this issue](https://github.com/Expensify/Expensify/issues/186628), and 2FA will be added with N7.